collectors/ebpf.plugin/ebpf_socket.c

```diff
@@ -2153,10 +2167,10 @@ void *ebpf_socket_read_hash(void *ptr)
     int fd_ipv4 = socket_maps[NETDATA_SOCKET_TABLE_IPV4].map_fd;
     int fd_ipv6 = socket_maps[NETDATA_SOCKET_TABLE_IPV6].map_fd;
     int network_connection = em->optional;
-    while (ebpf_socket_exited == NETDATA_THREAD_EBPF_RUNNING) {
-        usec_t dt = heartbeat_next(&hb, step);
-        (void)dt;
-        if (ebpf_socket_exited == NETDATA_THREAD_EBPF_STOPPING)
+    while (!ebpf_exit_plugin) {
+        (void)heartbeat_next(&hb, step);
+
+        if (ebpf_exit_plugin)
```

Comment:
Hey Thiago! Small one, maybe this if is not needed?

Reply:
I agree that was necessary with previous logic, I removed it with latest commit. Thanks!